tools/swarm/src/compose.rs

```diff
@@ -92,7 +92,7 @@ pub struct DockerComposeServiceBuilder {
     volumes: Vec<(String, String)>,
     trusted_peers: BTreeSet<PeerId>,
     genesis_public_key: PublicKey,
-    genesis_private_key: Option<PrivateKey>,
+    signed_genesis_file: Option<PathBuf>,
```

Comment:
_Probably, you haven't updated this part yet; if so, ignore this comment._

So, the genesis file remains the same, always unsigned. Just `genesis_file`. What this structure is lacking now is `genesis_signature` field, that should be represented as `GENESIS_SIGNATURE` env var.